autoload/neomake/makers/ft/purescript.vim

```diff
@@ -0,0 +1,59 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#purescript#EnabledMakers()
+  return ['pulp']
+endfunction
+
+function! neomake#makers#ft#purescript#pulp()
+    return {
+        \ 'exe': 'pulp',
+        \ 'args': ['build', '--no-psa', '--', '--json-errors'],
```

Comment:
`exe` defaults to `pulp`.
Is `--` required in `args` really?  (typically it's used to separate options from filenames, so `--json-errors` would end up being considered a filename?!)